### ISSUE TITLE
Refresh token should clear state, so it gets a new token from the endpoint

### ISSUE THREAD

**Issue by webbiscuit** [syl20lego/insomnia-plugin-aws-cognito-token#13]
This would be useful for testing users in different groups and changing attributes on the Cognito side. Right now we have to wait for the token to expire before it fetches a new one to pick up the server changes.

**Comment by syl20lego** [comment]
Good idea, I will look into it ... at the moment you can change the user or any other field (remove last letter and re-add it) that should cause a refresh.
